nbodykit/base/particles.py

```diff
@@ -119,9 +148,8 @@ def attrs(self):
         except AttributeError:
             self._attrs = {}
             return self._attrs
-
-    @staticmethod
-    def compute(*args, **kwargs):
+            
+    def compute(self, *args, **kwargs):
         """
         Our version of :func:`dask.compute` that computes
         multiple delayed dask collections at once
```

Comment:
I find myself frequently forget using this function! Shall we change the type of __getitem__ to ensure the .compute call fails?